docs/installation/autoware/docker-installation.md

```diff
@@ -1,13 +1,13 @@
-# Open AD Kit: Containerized Workloads for the Autoware
+# Open AD Kit: Containerized Workloads for Autoware
 
-Open AD Kit offers two types of Docker image to let you get started with the Autoware quickly: `devel` and `runtime`.
+Open AD Kit offers two types of Docker image to let you get started with Autoware quickly: `devel` and `runtime`.
 
 1. The `devel` image enables you to develop Autoware without setting up the local development environment.
 2. The `runtime` image contains only runtime executables and enables you to try out Autoware quickly.
 
 !!! info
 
-    Before proceeding, confirm and agree with the [NVIDIA Deep Learning Container license](https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license). By pulling and using the Autoware Open AD Kit images, you accept the terms and conditions of the license.
+    Before proceeding, confirm and agree with the [NVIDIA Deep Learning Container license](https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license). By pulling and using Autoware Open AD Kit images, you accept the terms and conditions of the license.
```

Comment:
Here it is "the ... images", so "the Autoware Open AD Kit images" was correct

Reply:
fixed in: https://github.com/autowarefoundation/autoware-documentation/pull/516/commits/19fd2966f325ad6937fe9ff09f586e3fdf5d3e49